x-pack/plugins/observability_ai_assistant/public/components/chat/welcome_message_connectors.tsx

```diff
@@ -32,6 +41,34 @@ export function WelcomeMessageConnectors({
   connectors: UseGenAIConnectorsResult;
   onSetupConnectorClick?: () => void;
 }) {
+  if (connectors.error) {
```

Comment:
This can happen when a user does not have access to the connectors, but does have access to the Observability AI Assistant (these are intentionally separate privileges).